packages/common/src/ui/layout/tables/components/Header/Header.tsx

```diff
@@ -58,8 +60,8 @@ export const HeaderCell = <T extends DomainObject>({
         borderBottom: '0px',
         paddingLeft: '16px',
         paddingRight: '16px',
-        width,
-        minWidth,
+        width: width + 10,
+        minWidth: minWidth + 10,
```

Comment:
Just put a placeholder icon for the button to resize right now and it needed some more room 🤷

Reply:
the current icon is, well, a little too bold for my tastes 😄

Reply:
Neither. Just a place holder to gauge reactions of spending a bit of time to do this!